app/services/risc_destination_updater.rb

```diff
@@ -24,27 +24,36 @@ def remove
     eventbridge_client.delete_connection(name: connection_name) if connection_exists?
   end
 
+  def issuer_slug
+    service_provider.issuer.gsub(/[^.\-_A-Za-z0-9]/, '_')
```

Comment:
perhaps some memoization?

Reply:
done in c4fa91c00ef46d11e4ee9b63bd5eb16c1a9b765e